README.md

```diff
@@ -1595,6 +1595,47 @@ _You can enable the following settings in Xcode by running [this script](resourc
 
 * <a id='newline-at-eof'></a>(<a href='#newline-at-eof'>link</a>) **Files should end in a newline.** [![SwiftLint: trailing_newline](https://img.shields.io/badge/SwiftLint-trailing__newline-007A87.svg)](https://github.com/realm/SwiftLint/blob/master/Rules.md#trailing-newline)
 
+* <a id='newline-between-sibling-scopes'></a>(<a href='#newline-between-sibling-scopes'>link</a>) **Sibling scopes should be separated by a newline.** Insert a single blank line between type, extension, protocol or function declarations at the same indentation level. [SwiftFormat: blankLinesBetweenScopes](https://img.shields.io/badge/SwiftFormat-blankLinesBetweenScopes-008489.svg)]
```

Comment:
```suggestion
* <a id='newline-between-sibling-scopes'></a>(<a href='#newline-between-sibling-scopes'>link</a>) **Sibling scopes should be separated by a newline.** Insert a single blank line between type, extension, protocol or function declarations at the same indentation level. [![SwiftFormat: blankLinesBetweenScopes](https://img.shields.io/badge/SwiftFormat-blankLinesBetweenScopes-008489.svg)](https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#blankLinesBetweenScopesPatternLet)
```

Reply:
Let's also add a Why? section

Reply:
This rule seems to apply to computed properties too, based on the draft PR that we created on our codebase. Can we indicate that somehow in the rule text?

Reply:
This rule seems to apply to computed properties too, based on the draft PR that we created on our codebase. Can we indicate that somehow in the rule text?

Reply:
I added a why section in b4af8265777b43e10be894cc172cb3918a099998